src/openzaak/conf/includes/base.py

```diff
@@ -591,6 +591,15 @@
     os.path.join(DJANGO_PROJECT_DIR, "notifications", "tests", "files"),
 ]
 
+#
+# VNG-API-COMMON
+#
+NOTIFICATIONS_DISABLED = config(
```

Comment:
you just added the setting to the `base` module, but didn't remove it from the `conf/ci.py` or `conf/dev.py` files.

I'd update the `ci.py` file to include `os.environ.setdefault("NOTIFICATIONS_DISABLED", "yes")` at the top, and I think we can leave the `dev.py` module alone.